CrossSectionHelper.py

```diff
@@ -89,7 +89,7 @@ class MCSampleValuesHelper():
         "SingleMuon_RunC": {
             "NEvents" : NEventsValues(
                 NEVT_UL16preVFP=67441308,
-                NEVT_UL17=165627777,
+                NEVT_UL17=165652756,
```

Comment:
Just to this is sure to be understood: this was a small number of events missing in the previous production that now ran through properly, right?